projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-search/comment-search.component.ts

```diff
@@ -47,6 +48,6 @@ export class CommentSearchComponent implements AfterViewInit, OnDestroy {
     this.searchString = undefined;
     this.searchResults = [];
     this.searchIndex = 0;
-    this.annotationEvents.resetTextHighlight();
+    this.store.dispatch(new fromStore.SearchComment(''));
```

Comment:
does this trigger the same actions as the reset highlight used to do

Reply:
yes it does